query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryResource.scala

```diff
@@ -410,7 +410,10 @@ class QueryResource(secondary: Secondary,
   private def parseFuseColumnMap(s: String): Map[String, String] = {
     s.split(';')
      .map { item => item.split(',') }
-     .map { case Array(a, b) => (a, b) }
+     .map { case Array(a, b) => (a, b)
+            case _ =>
```

Comment:
Maybe also throw the actual exception too.

Reply:
Or include the message from it on the new one.

Reply:
The actual exception is no match case in scala